File: 4Week/16Day/java/Algo07/src/test/two.java
package test;

public class two {
	public int solution(int a, int d, boolean[] included) {
		int len = included.length;
		int count = a;
		int answer = included[0] ? count : 0;
		
		
		for(int i = 1; i < len; i++) {
			count += d;
			answer += included[i] ? count : 0;
		}
		
		return answer;
    }
}

File: 4Week/16Day/과제/two.py
def solution(hp):
    '''
        장군개미 = 5
        병정개미 = 3
        일개미 = 1
    '''
    count = 0
    ant = [5, 3, 1]
    
    for i in ant:
        count += hp // i
        hp %= i
        
    return count
        